ArmaReforgerServerTool/Forms/Main.cs

```diff
@@ -75,6 +79,20 @@ public Main()
             Mod.GetScenariosForMod("591AF5BDA9F7CE8B");
         }
 
+        /// <summary>
+        /// Retrieves the application's version number.
+        /// </summary>
+        /// <returns>Formatted version string.</returns>
+        private string GetAppVersion()
```

Comment:
Probably shouldn't be in Main, maybe this can live in `FileIOManager`? There's not really a good home for it at the moment